src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -257,9 +260,33 @@
     </Compile>
     <Compile Include="System\Security\Cryptography\ECCng.ImportExport.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(TargetsWindows)' == 'true' AND ('$(TargetGroup)'=='' OR '$(TargetGroup)'=='netstandard1.7')">
```

Comment:
FWIW the' $(TargetGroup)'=='' is to get the files included in VS solution explorer... Perhaps to future-proof for 1.8 too, but for now it was for the VS editor.